src/SecTester.Core/Logger/DefaultConsoleFormatter.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Logging.Console;
+using Microsoft.Extensions.Options;
+using SecTester.Core.Utils;
+
+namespace SecTester.Core.Logger;
+
+public class DefaultConsoleFormatter : ConsoleFormatter, IDisposable
+{
+  private static readonly Dictionary<LogLevel, string> Levels = Enum
+    .GetValues(typeof(LogLevel))
+    .Cast<LogLevel>()
+    .ToDictionary(x => x, x => Enum.GetName(typeof(LogLevel), x));
+
+  private static readonly int MaxFormattedLevelLength = Levels
+    .Select(x => x.Value)
+    .OrderByDescending(x => x.Length)
+    .First()
+    .Length;
+
+  private static readonly Dictionary<LogLevel, string> FormattedLevels = Levels.ToDictionary(
+    x => x.Key, x => x.Value.ToUpper(CultureInfo.InvariantCulture).PadRight(MaxFormattedLevelLength));
+
+
+  private readonly IDisposable _optionsReloadToken;
+  private DefaultConsoleFormatterOptions _formatterOptions;

```

Comment:
I don't see any reason to couple them

```suggestion
  private ConsoleFormatterOptions _formatterOptions;

```